Assets/Scripts/ScriptableObjects/PlayerInformation.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using UnityEngine;
+
+namespace ScriptableObjects
+{
+    public class PlayerInformation : MonoBehaviour, Damagable
+    {
+        public PlayerInformation(double maxHealth)
```

Comment:
Although the issue mentions this as `PlayerInformation`, I'd argue this should be considered `PlayerHealth` for clarity. Thoughts @Migs ?